lib/active_model/serializer/associations.rb

```diff
@@ -92,7 +92,8 @@ def associations(include_tree = DEFAULT_INCLUDE_TREE)
 
         Enumerator.new do |y|
           self.class._reflections.each do |reflection|
-            next unless include_tree.key?(reflection.name)
+            name = reflection.options.fetch(:key, reflection.name)
```

Comment:
The tests say none. :-)

The beauty (and curse) of the test suite is that they'll catch the bugs for us (as long as we have the right tests)